Stylesheet/Components/grid.css

```diff
@@ -0,0 +1,15 @@
+/* This is grid file */
+
+.grid2 {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
```

Comment:
liked how your used repeat function instead of writing 1fr,1fr.

Reply:
Thank you